README.md

```diff
@@ -41,6 +41,7 @@ Using obj2gltf as a command-line tool:
 |`-o`|Directory or filename for the exported glTF file.|No|
 |`-b`|Output binary glTF.|No, default `false`|
 |`-s`|Writes out separate geometry/animation data files, shader files, and textures instead of embedding them in the glTF file.|No, default `false`|
+|`-p`|Use the optimization stages in the glTF pipeline|No, default `false`|
```

Comment:
Can you place this after the `-t` option? `-s` and `-t` make sense next to each other.